### ISSUE TITLE
Single element calculations when 2 are given give wrong results

### ISSUE THREAD

**Issue by Leimeroth** [ICAMS/calphy#60]
I calculated free energies for 2 element melts with different concentrations, including the end points, i.e. 0 and 100% of element A.
For convenience I thought I could simply use the same input file, but I noticed that at 100% of the second Element the result is wrong.
My guess is that this is related to the calculated concentration which does not state [0, 1] but just [1].
I assume this leads to the wrong mass being used for the reference phase and therefore wrong initial values, or something similar, since f over T plots look close to parallel, but start from completely different values.

TLDR: Using input files  for 2 elements with structures of concentration [0,1] leads to wrong results.

**Comment by srmnitc** [comment]
This should work. It is definitely a bug, I will look at it soon.

**Comment by srmnitc** [comment]
fixed in #108
